docs/r2/what-if-tool.md

```diff
@@ -0,0 +1,85 @@
+# Model Understanding with the What-If Tool Dashboard
+
+![What-If Tool](./images/what_if_tool.png)
+
+The What-If Tool (WIT) provides an easy-to-use interface for expanding
+understanding of black-box classification and regression ML models. With the
+plugin, you can perform inference on a large set of examples and immediately
+visualize the results in a variety of ways. Additionally, examples can be
+edited manually or programatically and re-run through the model in order to
+see the results of the changes. It contains tooling for investigating model
+performance and fairness over subsets of a dataset.
+
+The purpose of the tool is to give people a simple, intuitive, and powerful
+way to explore and investigate trained ML models through a visual interface
+with absolutely no code required.
+
+The tool can be accessed through TensorBoard or directly in a Jupyter or Colab
+notebook. For more in-depth details, demos, walkthroughs, and information
+specific to using WIT in notebook mode, see the
+[What-If Tool website](https://pair-code.github.io/what-if-tool).
+
+## Requirements
+
+To use WIT in TensorBoard, two things are necessary:
+- The model(s) you wish to explore must be served using
+  [TensorFlow Serving](https://github.com/tensorflow/serving) using the classify,
+  regress, or predict API.
+- The dataset to be inferred by the models must be in a TFRecord file accessible
+  by the TensorBoard web server.
+
+## Usage
+
+When opening the What-If Tool dashboard in TensorBoard, you will see a setup
+screen where you provide the host and port of the model server, the name of
+the model being served, the type of model, and the path to the TFRecords file
+to load. After filling this information out and clicking "Accept", WIT will
+load the dataset and run inference with the model, displaying the results.
+
+For details on the different features of WIT and how they can aid in model
+understanding and fairness investigations, see the walkthrough on the
+[What-If Tool website](https://pair-code.github.io/what-if-tool).
+
+## Demo model and dataset
+
+If you want to test out WIT in TensorBoard with a pre-trained model, you can
+download and unzip a pre-trained model and dataset from
+https://storage.googleapis.com/what-if-tool-resources/uci-census-demo.zip.
```

Comment:
Does this file exist already? I am not able to download it.

Reply:
fixed link